module/VuFind/tests/integration-tests/src/VuFindTest/Mink/FavoritesTest.php

```diff
@@ -729,23 +719,19 @@ public function testBulkDelete()
         // First try clicking without selecting anything:
         $button = $this->findCss($page, '[name=bulkActionForm] [name=delete]');
         $button->click();
-        $this->snooze();
         $this->checkForNonSelectedMessage($page);
 
         // Now do it for real -- we should get redirected.
         $this->selectAllItemsInList($page);
         $button->click();
-        $this->snooze();
         $this->clickCss($page, '.modal-body .btn.btn-primary');
-        $this->snooze();
         // Check for confirmation message
         $this->assertEquals(
             'Your saved item(s) were deleted.',
             $this->findCss($page, '.modal .alert-success')->getText()
         );
         $this->clickCss($page, '.modal .close');
-        $this->snooze();
-        $this->assertFalse(is_object($page->find('css', '.result')));
+        $this->unFindCss($page, '.result');
```

Comment:
...and then this failed consistently on the retries. None of the other tests before or after failed. Quite peculiar!

Reply:
I managed to reproduce after looping 15 times, but this is the one I encounter first. I had screenshots enabled, but since a screenshot was only taken after retries were exhausted, I didn't catch the initial problem yet. Now trying again with screenshots for every failure.

Retries fail trying to find the bulk delete button because the item was successfully deleted during the first round. No remaining items on retry, so no bulk buttons either. The retry logic doesn't quite work for a test case that depends on others. We may want to revise the tests to be more independent at some point, but that's for another time.